frontend/src/app/main/ui/shapes/text/fontfaces.cljs

```diff
@@ -73,16 +73,25 @@
     (when (d/not-empty? style)
       [:style style])))
 
+(defn frame->fonts
+  [frame objects]
+  (->> (cph/get-children objects (:id frame))
+       (filterv cph/text-shape?)
+       (mapv (comp fonts/get-content-fonts :content))
+       (reduce set/union #{})))
```

Comment:
If you finally go to reduce, don't use filterv and mapv

Reply:
ok, you're right